save-cloud-common/src/commonMain/kotlin/org/cqfn/save/user/UserRole.kt

```diff
@@ -0,0 +1,30 @@
+package org.cqfn.save.user
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Possible role users in project
+ */
+@Serializable
+enum class UserRole {
```

Comment:
We already have class `Role`, what's the purpose of the new one?

Reply:
Oh, good. I didn't see that such a class already exists